MAUI/Themes/HowTo.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: How to customize theming in Syncfusion controls
+description: Learn here about how to switch between light and dark themes in Syncfusion Maui controls and more details.
```

Comment:
description: Learn how to switch between light and dark themes in Syncfusion Maui controls and more details.